BUILD.gn

```diff
@@ -124,8 +123,12 @@ ts_sources = [
 rust_executable("deno") {
   source_root = "src/main.rs"
   extern = main_extern
+  data = [
```

Comment:
This is not a data dep.

Reply:
Use `inputs` or `sources`.

Reply:
Actually, it seems that the deno_exe.d depfile now include msg_generated.d - rust is smart enough to detect `include!()` it seems. I removed this.